forte/data/readers/openie_reader.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The reader that reads Open-IE extractions data into data pack.
+Format:
+https://github.com/gabrielStanovsky/oie-benchmark/tree/master/oie_corpus
+"""
+import codecs
+import logging
+import os
+from typing import Iterator, Any
+from forte.data.data_utils_io import dataset_path_iterator
+from forte.data.data_pack import DataPack
+from forte.data.readers.base_reader import PackReader
+from ft.onto.base_ontology import Sentence, PredicateMention, \
+    PredicateArgument, PredicateLink, Document
+
+__all__ = [
+    "OpenIEReader"
+]
+
+
+class OpenIEReader(PackReader):
+    r""":class:`OpenIEReader` is designed to read in the Open IE dataset.
```

Comment:
This docstring is not sufficient to describe the class. Imagine when a user is to use this library, when he pick class, what did he want to know? We need to make clear the file format, the expected input. It is also good to add the expected Entries that will be in the data pack after this reader. A better example of how to write documentation is here "https://github.com/asyml/forte/blob/master/forte/data/readers/ontonotes_reader.py#L35"